kafka-connect-cassandra/src/main/scala/com/datamountaineer/streamreactor/connect/cassandra/source/CassandraTypeConverter.scala

```diff
@@ -38,9 +38,9 @@ class CassandraTypeConverter(private val codecRegistry: CodecRegistry,
                              private val setting: CassandraSourceSetting) extends StrictLogging {
 
   val mapper = new ObjectMapper()
-  val OPTIONAL_DATE_SCHEMA = org.apache.kafka.connect.data.Date.builder().optional().build()
-  val OPTIONAL_TIMESTAMP_SCHEMA = Timestamp.builder().optional().build()
-  val OPTIONAL_DECIMAL_SCHEMA = Decimal.builder(18).optional().build()
+  val OPTIONAL_DATE_SCHEMA: Schema = org.apache.kafka.connect.data.Date.builder().optional().build()
```

Comment:
why did we add it back? we have the type inference

Reply:
Type annotations for public member is the standard.